tests/queries/0_stateless/03229_query_condition_cache_system_table.sql

```diff
@@ -1,41 +1,39 @@
--- Tags: no-parallel, no-parallel-replicas
+-- Tags: no-parallel
 -- Tag no-parallel: Messes with internal cache
 
+-- Tests system table 'system.query_condition_cache'
+
 SET allow_experimental_analyzer = 1;
 
 DROP TABLE IF EXISTS tab;
 
+CREATE TABLE tab (a Int64, b Int64) ENGINE = MergeTree ORDER BY a;
+INSERT INTO tab SELECT number, number FROM numbers(1_000_000); -- 1 mio rows sounds like a lot but the QCC doesn't cache anything if there is less data
+
 SELECT '--- with move to PREWHERE';
+SET optimize_move_to_prewhere = true;
 
 SYSTEM DROP QUERY CONDITION CACHE;
```

Comment:
Maybe it still makes sense to keep these 2 lines after clearing cache both times, wdyt ?

`SELECT 'Expect the cache to be initially empty.';
SELECT count(*) FROM system.query_condition_cache;`

Reply:
I guess the other test `03229_query_condition_cache_drop_cache` covers this situation already.